src/core/Core.js

```diff
@@ -783,25 +783,33 @@ class Uppy {
   /**
    * Logs stuff to console, only if `debug` is set to true. Silent in production.
    *
-   * @return {String|Object} to log
+   * @param {String|Object} msg to log
+   * @param {String} type optional `error` or `warning`
    */
   log (msg, type) {
+    let message = `[Uppy] [${Utils.getTimeStamp()}] ${msg}`
+
+    global.uppyLog = global.uppyLog + '\n' + 'DEBUG LOG: ' + msg
```

Comment:
This now always happens, even if opts.debug is false. Is that intentional?

Reply:
I thought maybe its useful that we can debug when needed my asking a user to send us `window.uppyLog` from console. But not sure, not feeling strongly about this.